InterviewFlashCards/AnswerViewController.swift

```diff
@@ -0,0 +1,146 @@
+    @IBOutlet weak var backButton: UIButton!
+    @IBOutlet weak var nextButton: UIButton!
+    @IBOutlet weak var answer: UILabel!
+    @IBOutlet weak var paginationLabel: UILabel!
+    
+    // MARK: Properties
+    var flashCard = IFCFlashCard()
+    var tempAnswerImageView: UIImageView?
+    var tempAnswerLabel: UILabel?
+    var isTapped = true
+    var leftGesture = UISwipeGestureRecognizer()
+    var rightGesture = UISwipeGestureRecognizer()
+    var tapGesture = UITapGestureRecognizer()
+    var index = 0
+    
+    // MARK: Lifecycle Methods
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        setupGestures()
+    }
+    
+    // MARK: Setup
+    func setupUI() {
+        paginationLabel.text = ""
+        setupTempImageViewBounds()
+        
+        if (flashCard.answerImages != nil) {
+            index = 0
+            answerImageView.userInteractionEnabled = true
+            answerImageView.image = flashCard.answerImages[0] as? UIImage
+        }
+        if (flashCard.answer != nil) {
+            answerLabel.text = flashCard.answer
+        } else {
+            answerLabel.text = ""
+        }
+    }
+    
+    func setupTempImageViewBounds() {
+        let width: CGFloat = answerImageView.bounds.size.width
+        let height: CGFloat = answerImageView.bounds.size.height
+        let x: CGFloat = answerImageView.bounds.origin.x
+        let y: CGFloat = answerImageView.bounds.origin.y
+        tempAnswerImageView = UIImageView(frame: CGRectMake(x, y, width, height))
+    }
+    
+    func setupGestures() {
+        let leftSwipeRecognizer: UISwipeGestureRecognizer = UISwipeGestureRecognizer(target: self, action: #selector(AnswerViewController.handleLeftSwipe(_:)))
+        leftSwipeRecognizer.direction = UISwipeGestureRecognizerDirection.Left
+        leftGesture = leftSwipeRecognizer
+        let rightSwipeRecognizer: UISwipeGestureRecognizer = UISwipeGestureRecognizer(target: self, action: #selector(AnswerViewController.handleRightSwipe(_:)))
+        rightSwipeRecognizer.direction = UISwipeGestureRecognizerDirection.Right
+        rightGesture = rightSwipeRecognizer
+        let tapGestureRecognizer: UITapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(AnswerViewController.handleRightSwipe(_:)))
+        tapGesture = tapGestureRecognizer
+        let pinchGesture: UIPinchGestureRecognizer = UIPinchGestureRecognizer(target: self, action: #selector(AnswerViewController.handleTap(_:)))
+        view.addGestureRecognizer(leftSwipeRecognizer)
+        view.addGestureRecognizer(rightSwipeRecognizer)
+        view.addGestureRecognizer(tapGestureRecognizer)
+        answerImageView.addGestureRecognizer(pinchGesture)
+    }
+    
+    func showPaginationLabel() {
+        paginationLabel.text = "\(index+1)/\(flashCard.answerImages.count)"
+    }
+    
+    func handleLeftSwipe(gesture: UISwipeGestureRecognizer) {
+        if index < flashCard.answerImages.count-1 {
```

Comment:
can you make this class more swift like? take advantage of `guard` and other swift constructs. This doesn't have to be a literal copy of the objc file in swift.